src/Blazored.Toast.TestExtensions/InMemoryToast.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Blazored.Toast.Services;
+using Bunit;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazored.Toast.TestExtensions
+{
+    public class InMemoryToast
+    {
+        private readonly ToastLevel _toastLevel;
+        private readonly RenderFragment _message;
+        private readonly string _heading;
```

Comment:
I would expose these as read only properties, to allow users to easily write their own assertions.